x/stakeibc/keeper/msg_server_register_host_zone_test.go

```diff
@@ -126,7 +126,7 @@ func (s *KeeperTestSuite) TestRegisterHostZone_Success() {
 	defaultMaxThreshold := sdk.NewDec(int64(stakeibctypes.DefaultMaxRedemptionRateThreshold)).Quo(sdk.NewDec(100))
 	s.Require().Equal(defaultMinThreshold, hostZone.MinRedemptionRate, "min redemption rate set to default")
 	s.Require().Equal(defaultMaxThreshold, hostZone.MaxRedemptionRate, "max redemption rate set to default")
-	s.Require().Equal(tc.unbondingFrequency, hostZone.UnbondingFrequency, "unbonding frequency set to default: 3")
+	s.Require().Equal(tc.unbondingPeriod, hostZone.UnbondingPeriod, "unbonding period set to default: 3")
```

Comment:
nit: doesn't look like it has a default? if so, can't find that, but prob isn't 3. would recommend we do NOT give it a default and fail the registration if a period isn't provided

Reply:
good catch, I think this is a stale description. Removed